Godeps/Godeps.json

```diff
@@ -373,12 +393,12 @@
 		},
 		{
 			"ImportPath": "github.com/elazarl/go-bindata-assetfs",
-			"Rev": "3dcc96556217539f50599357fb481ac0dc7439b9"
+			"Rev": "c57a80f1ab2ad67bafa83f5fd0b4c2ecbd253dd5"
```

Comment:
Thanks for this, godep save also backleveled logrus and golang.org/net but I guess we want our versions, especially for the net package.